WordPress/src/main/java/org/wordpress/android/ui/JetpackRemoteInstallFragment.kt

```diff
@@ -24,48 +22,35 @@ import org.wordpress.android.ui.accounts.LoginActivity
 import org.wordpress.android.util.AppLog
 import javax.inject.Inject
 
-class JetpackRemoteInstallFragment : Fragment() {
+class JetpackRemoteInstallFragment : Fragment(R.layout.jetpack_remote_install_fragment) {
     @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
     private lateinit var viewModel: JetpackRemoteInstallViewModel
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        with(JetpackRemoteInstallFragmentBinding.bind(view)) {
+            initDagger()
+            initViewModel(savedInstanceState)
+        }
+    }
+
+    private fun initDagger() {
         (requireActivity().application as WordPress).component()!!.inject(this)
     }
 
-    override fun onActivityCreated(savedInstanceState: Bundle?) {
-        super.onActivityCreated(savedInstanceState)
+    private fun JetpackRemoteInstallFragmentBinding.initViewModel(savedInstanceState: Bundle?) {
         requireActivity().let { activity ->
-            viewModel = ViewModelProvider(this, viewModelFactory).get(JetpackRemoteInstallViewModel::class.java)
-
             val intent = activity.intent
             val site = intent.getSerializableExtra(WordPress.SITE) as SiteModel
             val source = intent.getSerializableExtra(TRACKING_SOURCE_KEY) as JetpackConnectionSource
             val retrievedState = savedInstanceState?.getSerializable(VIEW_STATE) as? JetpackRemoteInstallViewState.Type
+            viewModel = ViewModelProvider(
```

Comment:
Minor (🔍): Consider splitting the lines by the dot (`.`) instead, like:
```
viewModel = ViewModelProvider(this@JetpackRemoteInstallFragment, viewModelFactory)
                    .get(JetpackRemoteInstallViewModel::class.java)
```

Reply:
@ParaskP7  - I ran the reformat code, I guess it didn't pick this up. 🤷

Reply:
👋 @zwarm !

Please treat it as minor, either do the change or not, I can live with it! 😄